### ISSUE TITLE
[Feature Request][Enchancement][UI/UX] HSV Color Selection option and Eyedropper

### ISSUE THREAD

**Issue by Abysmal2134** [Frooxius/NeosPublic#482]
It would be nice if it were possible to swap between RGB and HSV mode in the Color Selection wheel with the new UI. HSV is much more intuitive for faster and precise color adjustments rather than guessing how the RGB values would affect the color.

Another suggestion is having an extra button to spawn an Eyedropper tool so that you can pick colors from reference images (or the environment) with a pointer ray cast/laser that uses a camera to select a pixel's color values and automatically enter those values in the color wheel.

 I'm not sure if this is possible but Anomalous managed to make a chameleon texture to blend in with the environment by changing his hue, so assume there's a way to get the colors. Hope people find this suggestion useful!

**Comment by Frooxius** [comment]
Yes, I plan on reworking the color dialog with UIX and have it include things like this! RGB and HSV modes, hex codes, color picker and such!
